cli/module_registry.go

```diff
@@ -117,20 +126,33 @@ func CreateModuleAction(c *cli.Context) error {
 	if response.GetUrl() != "" {
 		printf(c.App.Writer, "You can view it here: %s", response.GetUrl())
 	}
-	emptyManifest := moduleManifest{
-		ModuleID:   returnedModuleID.String(),
-		Visibility: moduleVisibilityPrivate,
-		// This is done so that the json has an empty example
-		Models: []ModuleComponent{
-			{},
-		},
-		// TODO(RSDK-5608) don't auto populate until we are ready to release the build subcommand
-		// Build: defaultBuildInfo,
-	}
-	if err := writeManifest(defaultManifestFilename, emptyManifest); err != nil {
-		return err
+
+	if modManifest != nil {
+		// if the original module manifest we read was non-nil, we also want to run an update
+		_, err := client.updateModule(returnedModuleID, *modManifest)
+		// we want to fail with a warning rather than an error because this isn't an action the user asked for,
+		// so red scary error messages would cause confusion
+		if err != nil {
+			warningf(c.App.Writer, "Tried to update module with info from your meta.json but got: %v", err)
```

Comment:
could also silently fail (or just not update... but then people /will/ be confused about why the info isn't on the webpage they were told to visit and which is in the meta.json file for the module they just created.)

Reply:
This is fine by me. I'm curious to see what users will naturally encounter. I guess if they clear out descriptions, models, etc.